### ISSUE TITLE
Add support for SSMS 2017 ?

### ISSUE THREAD

**Issue by datauduong** [bugzinga/sql-hunting-dog#7]
Please Add Support for SSMS 2017, Thanks in Advance.

**Comment by darlove** [comment]
Yeah, it would be super cool if we could have support for the latest SSMS...

**Comment by KyleMit** [comment]
One added component in supporting newer versions is that SSMS 2010 / 2012 used "Add-Ins" which have been deprecated and replaced with "VSIX projects".

Here's a migration guide on [Converting add-ins to VSPackage extensions][1]

But it's not a trivial lift or simple config change + republish

[1]: https://docs.microsoft.com/en-us/visualstudio/extensibility/faq-converting-add-ins-to-vspackage-extensions?view=vs-2019
